.github/ISSUE_TEMPLATE/enhancement_template.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Enhancement Request
+about: Suggest an idea for Kolibri Studio
+title: "[Title]: Brief description"
+labels: ''
+assignees: ''
+
+---
+
+<!-- Please remove any unused sections.
+
+Note that anything written between these symbols will not appear in the actual, published issue. They serve as instructions for filling out this template. You may want to use the 'preview' tab above this textbox to verify formatting before submitting.
+
+Instructions:
+- Title line template: [Title]: Brief description
+
+-->
+
+## Desired behavior
+<!-- Briefly describe the behavior you would like to see -->
+
+
+## Current behavior
+<!-- Briefly describe the current behavior; you may include screenshots, code, and notes -->
+
+
+## (Optional) The Value Add
+<!-- Explain why this should be added or changed in KDS and where it could be used -->
+
+
+## (Optional) Possible Tradeoffs
+<!-- Explain possible issues/costs that could arise - if any - from implementing this enhancement -->
```

Comment:
minor but in the spirit of consistency ... I think LE is slowly moving towards `Sentence case` rather than `Title Case` headers in general... It's in the design system but we haven't "enforced" it outside of product text.